crates/diesel_models/src/payment_method.rs

```diff
@@ -64,6 +64,7 @@ pub struct PaymentMethod {
     pub network_token_requestor_reference_id: Option<String>,
     pub network_token_locker_id: Option<String>,
     pub network_token_payment_method_data: Option<Encryption>,
+    pub transaction_flow: Option<storage_enums::PaymentDirection>,
```

Comment:
For the previous records where this would be not be there. 
How would you be processing those entries? Or would those remain untouched for this flow?

Reply:
Yes, those entries will be untouched for this flow. This field should later be consumed in payment flows, to filter payment methods for payments vs payouts.